css/src/utils/utility.css

```diff
@@ -34,3 +34,202 @@
 .bottom-0 {
   bottom: 0;
 }
+
+/** New utility classes **/
+
+/* Visibility */
+.visible {
+  visibility: visible !important;
+}
+
+.invisible {
+  visibility: hidden !important;
+}
+
+/* User select */
+.user-select-none {
+  user-select: none !important;
+}
+
+.user-select-all {
+  user-select: all !important;
+}
+
+.user-select-auto {
+  user-select: auto !important;
+}
+
+.user-select-text {
+  user-select: text !important;
+}
+
+/* Word break */
+.word-break-normal {
+  word-break: normal !important;
+}
+
+.word-break-all {
+  word-break: break-all !important;
+}
+
+.word-break-word {
+  word-break: break-word !important;
+}
+
+.word-wrap-break {
+  word-wrap: break-word !important;
+}
+
+/* List style */
+.list-style-none {
+  list-style: none !important;
+}
+
+.list-style-disc {
+  list-style: disc !important;
+}
+
+.list-style-circle {
+  list-style: circle !important;
+}
+
+.list-style-square {
+  list-style: square !important;
+}
```

Comment:
Some of the other classes for list-style are missing like (auto, decimal, lower-alpha, upper-alpha) etc.